FX/BryanChi_FX Devices.lua

```diff
@@ -1554,10 +1638,15 @@ function GeneralFunctions()
     end
 end
 
+function CopyFile(old_path, new_path)
+    local CopyCmd = r.GetOS():find("Win") and 'copy "' or 'cp "'
+    local ok = os.execute(CopyCmd .. old_path .. '" "' .. new_path .. '"')
```

Comment:
This code is dangerous: generating a command line from outside data to be executed by the system shell without proper sanitization.

Copying files can be done without leaving Lua: https://forum.cockos.com/showthread.php?p=236058.

Reply:
Thanks cfillion, I've updated the function with the one you provided, all seems to be working well!